tests/Fixer/Phpdoc/PhpdocIndentFixerTest.php

```diff
@@ -381,6 +381,16 @@ public function bar()
 ',
         );
 
+        $cases[] = array(
+            '<?php
+
+function foo()
+{
+    $foo->bar(/** oops */$baz);
```

Comment:
Works just fine. I tested that earlier. Probably worth adding it as a unit test though. 👍